modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -76,6 +77,32 @@ A clone of the Git repository is present in the filesystem of the new workspace.
 +
 The workspace has a unique URL: `pass:c,a,q[{prod-url}]#workspace__<unique_url>__`.
 
+.. Use a direct URL to `devfile`:
+[source,subs="+quotes,+attributes,+macros"]
++
+----
+pass:c,a,q[{prod-url}]#__<url_to_devfile>__
+----
++
+[TIP]
+====
+You can pass Personal Access Token to the URL to have access to private repositories:
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__https://__<token>__@__<host>__/__<path> <1>
```

Comment:
@vinokurig probably here we need to add `IMPORTANT:` note that devfile should contain the project reference, also maybe worth mentioning that path = path to devfile